pybossa/exporter/consensus_exporter.py

```diff
@@ -133,6 +133,7 @@ def format_consensus(rows):
                     if user_id not in local_user_cache:
                         user_info = get_user_info(user_id) or {'user_id': user_id}
                         local_user_cache[user_id] = user_info
+                    user_info = local_user_cache[user_id]
```

Comment:
Consider a minor optimization with an if/else block, to avoid overwriting `user_info` when populating the first time.

```python
if user_id not in local_user_cache:
    # Fetch user info from db and store in cache.
    user_info = get_user_info(user_id) or {'user_id': user_id}
    local_user_cache[user_id] = user_info
else:
    user_info = local_user_cache[user_id]
```

Reply:
sure, will update the PR

Reply:
would 
```
 if user_id in local_user_cache:
        user_info = local_user_cache[user_id]:
else:
        # Fetch user info from db and store in cache.
        user_info = get_user_info(user_id) or {'user_id': user_id}
        local_user_cache[user_id] = user_info
```
make it a happy path?

Reply:
Sounds good, thank you.